_docs/flavour/templates.md

```diff
@@ -74,7 +74,7 @@ public static void main(String[] args) {
 }
 ```
 
-Of course, your master `index.html` should contain element with `application-content` identifier.
+Of course, your master `index.html` should contain element with `application-content` id.
```

Comment:
I thought it would be clearer to use the exact attribute name to avoid any confusion.  Maybe it would be best to include both.  What do you think?
 
* contain an element with 'application-content' identifier (id="application-content").